src/Stream.re

```diff
@@ -58,6 +65,7 @@ let map = (v: t('a), f: 'a => option('b)) => {
       | Some(v) => send(v)
       }
     )
+    |> (ignore: unsubscribeFunc => unit)
```

Comment:
Not 100% sure about this one, but at least this won't affect any existing functionality.